### ISSUE TITLE
Bunch a things

### ISSUE THREAD

**Issue by Au-heppa** [loiste-interactive/infra-issues#74]
**THE CONS:**

**1) Having to leave game and load the main menu map is completely nonsensical** 
Every time I wanted to tweak the mouse sensitivity I had to load the map again. I had to load the map four times. No 3d main menu is impressive enough to warrant this. If you really want to keep it, just have a separate options menu when you are in-game.

**2) When I go out of the game to, say, leave feedback and then try to go back the screen goes black** 

**3) No map** 
Not really 100% necessary but would be appreciated at least in the first level. Would help a lot with aimless wandering. Aimless wandering is only fun if there's something kind of reward for it. There is none. You already have a flashlight and the player character is established is as not-bodyless spirit so having a map would not feel out of place. Also would make sense the guy would have a map of the place he has come to work in.

**4) No voice overs. No story? What is this game about?**
Again. If having a story or voice over is not your bag I can totally get it, but right now I am not getting any kind of story from the game (though I am not very far). Having a story is not a requirement for a game either, but usually then there's something else (puzzles or other gameplay). I haven't played Proteus though and I understand that game has no gameplay or story per se, just exploration. That is all fine, but I don't get a huge feel of exploration though. So far I am not getting a huge enjoyment from exploring the world, I don't get a sense of wonderment of "what's there? what's that?"

Adding some kind of story this late would be a bad idea, probably though, if you are not in the process of adding it already. Also, I am not sure what the story would be?

**5) No saving**
This is another nonsensical decision. If am playing the game, I finish a map I am going to think hard if I have energy to and or time to play another map. Where as normally I would start the map and then save and quit when I need to here I am going to probably quit the game if I have the slightest feel I wont have time to finish the level.

I am not sure if I saw quicksaving mentioned in the options but I did try pressing F6 in-game. Since there was no text for it it looked like it didn't save. But later on when I tried to smash F6 it seemed to lag a bit like if it was saving so maybe it does? I am not sure. I didn't try pressing quickload in the mainmenu yet (either quickloading wont work or the command is blocked by the menu).

Also, I have no idea what map is what from the menu yet. So, I played the first map and it loaded the next one after the elevator? Is that the second option from the new game menu or will I have to replay the first map if the quickloading wont work?

**6) Sluice generator puzzle meters are unreadable**
The meters for the sluice generator puzzle are pretty much unreadable from the standard eye position. I have to crouch every time to read them which makes it really tedious. They should be on the eye level.

**7) Electrical water boxes**
Before quitting to eat my dinner I was at the electrical water puzzle. I tried carrying the boxes for a half-life-esque path across the water. But the boxes didn't have glow on them like every other object (made me think I wasn't able to pick them up at first), and some of the boxes were completely static for no apparent reason.

**8) No HUD / Some of the default controls are odd**
I had the same problem with my own game where people would always try to use +attack instead of +use for picking stuff up or using some machines. It's kind of a petpeevee but I did have the trouble of remembering to press +use instead of +attack to use most items / objects.

**9) No rewards**
In games like Portal or The Vanishing Of Ethan Carter you have rewards from finishing or getting into another section. In the latter it encourages exploration since you don't feel like you have wasted time if you go to the "wrong" direction. I am not sensing any kind of reward from going to the wrong direction or finishing a puzzle. Again, the concept of "rewards" is not necessary for a game.

**THE PROS**

**1) It looks pretty**
Though to be kind of dick-ish I would add "for a Source game". I am not sure if this would be a selling point alone unlike for games like The Vanishing Of Ethan Carter or Everyone's Gone To Rapture.

**2) No hud**
It fits the style.

**SUMMARY**
So far I am not sure what the game is supposed to be. At first I thought it was going to be Gone Home or The Vanishing Of Ethan Carter with voice overs and a story, but it doesn't look like that so far. What I have understood so far the story is about collapsing buildings / infrastructure. I am not sure how that will become a human story. 

_Gone Home_ works for me because it's an understandable human story about human beings. 
_Dear Esther_ doesn't work for me because it's pretentious and I have honestly no idea what the story was about. 
_The Vanishing Of Ethan Carter_ worked, sort of, since the story was understandable, the cut scenes were enjoyable (even if the story ended up being trash).

I am curious, though, dubious, what the story of INFRA will be if there is any.

If it's not a story game then is it a puzzle game? There have been some very linear "press button" type of puzzles so far, but nothing interesting yet. My question is that how interesting can the puzzles get with this system? What's the twist? Portal has, well, portals. The Vanishing Of Ethan Carter had a lot of different stylistic puzzles (though I wasn't too stoked about them).

Again, I am not very far in the game, so a lot of my story and gameplay issues might be proven completely wrong, or maybe there's something I am just not getting yet. I will comment more after I have played more.

**Comment by Au-heppa** [comment]
Hurray, there was a little bit of story! Also, quicksaving worked, hurray. I am not sure if any player that is not familiar with Source would know of quicksaving though. I also wasn't able to use quickload in the menu, instead I had to first load the first map and then press quickload. Unless this is intentional, in Source code for panels, you can pass through commands like "+quickload" "screenshot", etc. through the panel input system to allow those commands even if the panel is selected.

**CONS CONTINUED**

**10) Water is invisible in electric water room** 
So I continued with my crate shenanigans. Placed a bunch of crates on the water so I could first walk over the line of electric boxes and then on the crates to the stairs. I put one crate on the water near the cylinder thing, and I die. There was no way to tell where the water level was so I couldn't tell that one crate wouldn't be enough. Not sure if I would have had to restart the map if it wasn't for the quicksave. That would had not been fun. Anyway, I tried again by stacking two crates instead of one.

**11) Basement and the mystical +attack key**
I could be wrong but it seemed like I randomly had to use +attack instead +use in the section with the basement electrical button. The invisible wall blocking from going through the generators was also a bit weird. What was super annoying though was continuously dying from the electricity. I carried the crate from the part where I had to climb to get the keys (the only crate that could be picked up, even though it's relatively same size as all the other crates). I tried placing it on the water to press the button, but wasn't able to escape because of the invisible wall on the generators.

It took me back all the way to to beginning of that section (before even the key puzzle). I am starting to get the impression this game would be really frustrating without the quicksaving (still is with it).

**Comment by aleksijuvani** [comment]
> 1) Having to leave game and load the main menu map is completely nonsensical

I agree. I need to add options to the pause menu. There's issue #16 to track this.

> 2) When I go out of the game to, say, leave feedback and then try to go back the screen goes black

I'm not sure what in the world could be causing this. It works fine with all three display modes on my computer (fullscreen, borderless and windowed). The same issue was reported in #42.

> 5) No saving

We've disabled saving in the beta. Well, it's still there, but there's no way to load saved games through the menu. Saves break if we make any changes to the maps, so I felt like we could just unlock all of the maps and disable saving.

It's kind of shit. I'd hoped I'd find a better solution than this.

I suppose it's kind of weird how only the first map of each chapter is available in the main menu though. I wonder if just adding every map there would remedy this.

> 8) No HUD / Some of the default controls are odd
> I had the same problem with my own game where people would always try to use +attack instead of +use for picking stuff up or using some machines. It's kind of a petpeevee but I did have the trouble of remembering to press +use instead of +attack to use most items / objects.

This did cross my mind earlier. What we could do is have MOUSE1 be the "use" button when you don't have the camera equipped, and be the "take picture" button when you do have the camera equipped. What do you think?

**Comment by Au-heppa** [comment]
Oh yeah, I just discovered that I have a camera one second ago. I had no idea I had it and I am not sure what I am supposed to use it for. I was going suggest something like having the player character sometimes bring a notepad and scribble something to  give a sense of story, since he or she didn't seem to do anything relating to this job while there. 

But since the game didn't give me any kind of instructions I am a bit confused.

I found out about the camera when I was trying to check if there was zoom key like in HL2. I found a pile of papers and when I pressed attack or use the character brushed some of the papers aside, but it's impossible to read. I tried crouching but then my nose is in the same level as the papers. Not sure if there has been any other papers with story stuff in previous levels, but I sure missed them because I had no idea I could interact with papers. The game seems a bit random when it gives you the glow for interactive objects and when not. Doesn't seem to matter if I can't read the papers.

Also, yes, using +attack instead of +use would be better. I sort of have the same problem in my own project.

Also, yes, the saving problem does make sense. I have been trying to figure out a good solution since people can't really stop Steam from loading updates. Changing the map is ok if you don't change the layout too much though (So that they wont end up inside a wall). Changes to entities wont be applied to the save files. If you delete any brush entities the game will give you an error message and wont load. But if you for example, add a new battery entity somewhere in your map and recompile, or tweak the lighting and recompile, it wont cause problems.

**Comment by Oskutin** [comment]
For the story:  There's story in the game which advances in multiple levels and tells about events in multiple viewpoints. Every level is literally cluttered with documents from the past, but noticing them may need some attention to detail.  We left some keypoints of the story unimplemented/partially implemented in the beta.  Also as we left the first level (briefing) and two from the end out of the beta, it may explain some part of feeling "lost". We may add them little later to beta though.


Sorry, I can't explain more in detail as i'm in hurry now.

**Comment by aleksijuvani** [comment]
> Oh yeah, I just discovered that I have a camera one second ago. I had no idea I had it and I am not sure what I am supposed to use it for. I was going suggest something like having the player character sometimes bring a notepad and scribble something to give a sense of story, since he or she didn't seem to do anything relating to this job while there.

Oh man, I really need to add some kind of tutorial messages for these.

> I found out about the camera when I was trying to check if there was zoom key like in HL2. I found a pile of papers and when I pressed attack or use the character brushed some of the papers aside, but it's impossible to read. I tried crouching but then my nose is in the same level as the papers.

You should be able to use MOUSE3 to zoom if you're using the default controls.

**Comment by Au-heppa** [comment]
Oh yeah, I did try the mouse3 but it's built into the mousewheel which essentially makes mouse3 useless (when I try to press mouse3 70% of the time I press mousewheel up or down instead). So I didn't realize there was zoom button. Again, this would be no problem if I could just go to options while running the game.

Still no idea what the camera is for. Still dying to the electricity.

**Comment by aleksijuvani** [comment]
You're in the flooded basement room with the electrical switch, right? You don't need a crate there, all you need to do is flip the switch and run back to the stairs.

**Comment by Au-heppa** [comment]
Using windowed mode fixed the black screen problem. Should have switched to that sooner.

Also yes, now that I figured I can read things I am getting a much better sense of story. There's still places where the text on a wall is not in the eyelevel making it really awkward to try to read. Also, are you guys using rendertextures for the notes and stuff? I always wondered how something like Gone home would handle localizing all the notes since they were handwritten.

**Comment by aleksijuvani** [comment]
> Also, are you guys using rendertextures for the notes and stuff? I always wondered how something like Gone home would handle localizing all the notes since they were handwritten.

Nah, we're using plain textures. We thought about all kinds of weird systems to handle localization here, but it turns out that you can just localize the textures.

Anyway, thanks for taking the time to write down all this feedback. It's really helpful to get an additional perspective on things. When you work on something for this long, it's easy to just lose sight of all that is there.

**Comment by Au-heppa** [comment]
Even with the zoom it's sometimes difficult to read the text... I would suggest having a similar to SOMA where you get an option for UI version for the text (you could even save time localizing the textures by having the localized version as UI version). I am not sure if it will be possible to read the text with every resolution and with lowered texture quality (though I guess you can just disable LODs / mipmap).

**Comment by aleksijuvani** [comment]
At one point we had a system where you could use the notes and the camera would transition really close to the note so that they'd be easier to read. It was before we had use highlights so it seemed really inconsistent and confusing. We also never finished fully implementing it, so we removed it at some point to get a playtest build out.

This might be worth revisiting. It could be complemented by adding a button for displaying a UI version of the text, but I'm not sure how this would fit with our "no HUD" philosophy.

**Comment by Au-heppa** [comment]
Gone Home didn't have UI text, I think, and it worked for them. Been a while since I played that though. Zooming would work as long as you tweak the fadein fadeout so that it doesn't feel awkward (or that it takes too much time).

**Comment by Au-heppa** [comment]
Also, I just died because I didn't know the floodgates had to be open in a correct order and didn't know what to do after trying the closest valve and it didn't open.

I also kind of wish there was a lot more sequences with big machinery doing their thing (the bridges don't count).

**Comment by Au-heppa** [comment]
I think things not being on the player eye level is my number one complaint right now. 

I can still notice the buttons but it's super annoying to look down and not be able to see the text or different parts properly. The Sluice generator puzzle meters thing was the worst since you really needed to be able to read the values. I would also probably make them bigger. It might be tricky to have to redo the uv mapping and what have you but it will be worth it.

Not having crosshair can be annoying as well since it can be like trying to find your way in the dark when trying to select the right button among many.

**Comment by Au-heppa** [comment]
I get the feeling like maybe there's some easter eggs or secret areas that I am missing.

In the previous level there was the box that I had to open with the keys, and it had these four on/off switches with red green markins, and one big on and off button, and one light for sensor or something and one for the on/off. I couldn't figure out what it was supposed to be and if it was related to the bridge, so maybe it was like a secret thing?

I previously I found a note about valves or something and "how they are here somewhere" or something like that. I found a bunch of valves and started to carry them around. But because I couldn't put them in my inventory I could only carry one at a time which was tedious. There was also a bunch of valves attached to the pipes, some of which seemed to be randomly static, some of them seemed to be interactive, but not yet possible to turn. I wasn't too thrilled about all this so I was getting ready to quit for today, but then randomly found the exit, finding out that I didn't need any of them.

If this is some kind of secret or bonus area with more story stuff, I am not sure if I am correctly motivated right now. There hasn't been a whole lot of story so far, so I am not hooked on finding more. The only bits of story I have found so far are "rich guy lost his money, sadface" and little bits about the workers being as confused as I am. Maybe there is a bigger story here, but I might have missed it because I didn't even know about the papers until later on.

Also, I have no idea where I would have had to put the valves or what I would get from finishing the supposed puzzle so it's hard to give myself the motivation. 

I guess it's possible that the level had no puzzles at all and I am just thinking the valves were there for something because of video games.

**Comment by Au-heppa** [comment]
Raft thing that starts moving the minute you step on it. Jump right before it hits the fall. Get lodged inside the wall. Unable to die. Unable to call for friend. No phone. No food. Quicksave was from previous map. Send help.

>:P

All in all, even with my negative comments, I have been enjoying it. It's a very pretty game. Will probably play more today or tomorrow.

**Comment by aleksijuvani** [comment]
>  There was also a bunch of valves attached to the pipes, some of which seemed to be randomly static, some of them seemed to be interactive, but not yet possible to turn.

Oh yeah. This is kinda stupid. For some reason we have this pipe model with a small red valve on it, but it's just a static prop without any gameplay purpose. These are all over the levels. I think it will get better once everything that can be used has use highlights, but it might also be worth considering replacing the model.

**Comment by mpeltomaa** [comment]
About that mouse wheel, I play games sometimes with laptop without mouse. I started to play Infra that way also. So, there is no wheel or button3. This should be handled somehow because there is lot of people who don't even own mouse and are still playing games a lot.

**Comment by aleksijuvani** [comment]
> About that mouse wheel, I play games sometimes with laptop without mouse. I started to play Infra that way also. So, there is no wheel or button3. This should be handled somehow because there is lot of people who don't even own mouse and are still playing games a lot.

Makes sense. I could add an alternative key for the zoom. You should be able to change it in the options menu for now.

**Comment by Au-heppa** [comment]
I went for "z" when trying to check if there was one since that's zoom in HL2 by default, I think.

**Comment by Au-heppa** [comment]
So, should I continue from where I left off or wait until you have done some updates? (I am fine with just continuing where I am, but I can also wait and then restart once there's some updates out).

I also think that leaving the intro out might be really detrimental. Not sure if that's by choice or because you haven't finished it yet? But I have no idea about what the story is and if the story is big part of the experience I can't really have a full experience. I understand if this is supposed to be a play test for just the puzzles, but you should test story impact, too.

Also, wondering, does the camera record anything, save any info about what you have taken pictures of? I haven't tried taking a lot of photos because I have no idea what it does. I am wondering if there's some dialogue when I take a photo, or if it's something like in Life Is Strange where you "collect" the photos of important things, or if it's just something for player can do to get into the role of the character but has no gameplay implications. By gameplay here I mean anything where the game gives you feedback for pressing the take photo button even if it's just little bit of dialogue (not something that is necessary).

Also, in hindsight the complaint about no map seems obsolete. It seemed like it could have been something that would be good in the first map when I thought all of the maps were going to be bigger exterior areas, but a lot of the later maps seem to be a lot more compact so there hasn't been a lot of backtracking at least not as much as in the first level.

I forgot to mention it before, but in the first map there was a bunch of cabins / houses that I really liked and I was going to make a comment about how I wanted a game that was all about that Finnish summer cottage / old style house aesthetic. I took a screenshot of the house, but I wasn't sure if it was OK for me to upload it anywhere so I didn't.

**Comment by aleksijuvani** [comment]
> So, should I continue from where I left off or wait until you have done some updates? (I am fine with just continuing where I am, but I can also wait and then restart once there's some updates out).

Completely up to you! I was sort of half-joking on Twitter. I'm not sure when the next update will be out, but we've fixed some issues in the last one. The patch notes are [here](https://github.com/loiste-interactive/infra-issues/blob/master/CHANGES.md#november-3rd-2015).

> I also think that leaving the intro out might be really detrimental. Not sure if that's by choice or because you haven't finished it yet? But I have no idea about what the story is and if the story is big part of the experience I can't really have a full experience. I understand if this is supposed to be a play test for just the puzzles, but you should test story impact, too.

I agree. We haven't finished it yet. We're probably including it in the beta once it's done.

> Also, in hindsight the complaint about no map seems obsolete. It seemed like it could have been something that would be good in the first map when I thought all of the maps were going to be bigger exterior areas, but a lot of the later maps seem to be a lot more compact so there hasn't been a lot of backtracking at least not as much as in the first level.

There actually is a map of the area pinned to the wall of the logging cabin in the first level. It wasn't very useful, but we've improved it in the next update. You can't take it with you, but it should give you an idea of where to go.

> Also, wondering, does the camera record anything, save any info about what you have taken pictures of? I haven't tried taking a lot of photos because I have no idea what it does. I am wondering if there's some dialogue when I take a photo, or if it's something like in Life Is Strange where you "collect" the photos of important things, or if it's just something for player can do to get into the role of the character but has no gameplay implications. By gameplay here I mean anything where the game gives you feedback for pressing the take photo button even if it's just little bit of dialogue (not something that is necessary).

When the player can take a successful photo, the camera "focus rectangle" will "zoom in". When you take a photo of these spots, a sound is played to indicate a successful photo. The amount of successful photos is saved (as is the amount of spots repaired and mistakes made).

All these things will eventually affect the ending of the game. We've thought about adding ~~dialogue~~ monologue to taking photos of important spots, such as the bridge and the dam in the first level.

> I forgot to mention it before, but in the first map there was a bunch of cabins / houses that I really liked and I was going to make a comment about how I wanted a game that was all about that Finnish summer cottage / old style house aesthetic. I took a screenshot of the house, but I wasn't sure if it was OK for me to upload it anywhere so I didn't.

Oh, screenshots and videos are fine. Go for it!

**Comment by Au-heppa** [comment]
If the number of photos you have taken affects the game then not being able to save is a problem since it wouldn't be loaded when you continue from another map using the new game menu rather than quick load.

I would suggest re-enabling the load and save menus and only pushing updates on Sunday evenings or something. Let everyone play tester know that this is the case and that if they continue playing a save file from the previous week they risk having bugs and problems that you have already sorted out. Make it really clear that this is the case. It might feel like it's slowing the progress a little bit, but it's better for the play testers.

In my own game I had a lot of problems where I decided to fix the layout of the map or change the puzzles completely and I had to ask if it was OK for them to have to restart playing the game. Everyone seemed to be OK but I suppose after constant fear of updates breaking their save files they stopped playing the game completely. I haven't had play testers play it in a while now. So, I can sort of understand not wanting to include the load files, but it can also just make things more complicated for play testers. 

Using Steam to play test your game really doesn't seem like the best option. I should have just given everyone access to the SVN repository instead. Steam is kind of a trade off because you don't have to ask people to jump through all these whoops (download Source SDK Base 2013 (My repository version doesn't include all the engine stuff the Steam version does and thus requires you to download the SDK), install tortoise SVN, get an username and password the repository, download the game, deal with possible TortoiseSVN problems (like the download calling because of server error) or conflict in files. 

Steam makes it all one button and go, but the problem seems to be that it's no longer possible to disable the game from updating. With TortoiseSVN the play testers could download the update when they want (but it can also mean they forget to do it and end up playing an old version) so they can play through the whole game with just one version.

Also, here's the cabin I was talking about:

![2015-11-02_00001](https://cloud.githubusercontent.com/assets/5713546/10995390/1bcd1d72-8485-11e5-87d9-af3739e96edb.jpg)

**Comment by Au-heppa** [comment]
Okay, played some more:

Why is quick load F7 by default? I thought it was F9 in HL2 and other Source games. If it's F7 it's very easy to accidentally press F6 to quick save instead.

You can reload the flashlight?

Okay, so I got to the boat part again. As the boat starts moving the minute you step on it, they are standing right on the edge of the thing and it looks like they are standing on air. This time I figured you can actually direct the boat. Not too thrilled about the controls. At least since you are right at the back of the boat you can see the edges of the boat without having to look down, but still it's very hard to see where you are in relation to a wall a lot of time. Sometimes in addition to having no depth reception it's way too dark to see even with the flashlight full of new batteries.

The boat ride doesn't seem too exciting right now either. There are no any kind of water effects. The boat doesn't make any kind of sounds or splashing effects. If I couldn't constantly see the boat under me it would feel like I was just a ghost floating above water. The player character seems to be standing a bit unnaturally on the boat any way, considering he's trying to control it. Having no sense of your body being affect by the boat rocking would feel more natural if I wasn't required to control the boat.

I do appreciate that the game instantly loads after you die, but it's still kind of frustrating and at the same time very uneventful (there is no sound, no animation, the boat just breaks into pieces in one go). A lot of times I thought I was supposed to land on one of the beaches, but instead I just broke my boat few inches from the land and died. Bunch of times I was going the wrong way either because it was too dark or because it looked like the right direction first, by the time I realized my mistake it was too late and I crashed my boat again. I am not sure if the player should be able to die.

Right now it feels like an oddity and a huge change from what the game has been so far. It's like a slowed down bullet hell game where your hitbox is not one pixel size like usual.

Oh yeah, I thought this place was really pretty:

![2015-11-06_00001](https://cloud.githubusercontent.com/assets/5713546/10996619/6c19b452-848f-11e5-9972-ea93338f2f51.jpg)

**Comment by aleksijuvani** [comment]
> Why is quick load F7 by default? I thought it was F9 in HL2 and other Source games. If it's F7 it's very easy to accidentally press F6 to quick save instead.

I can't remember if I've changed it. I don't think that I have. If I have though, it might be worth changing back.

> You can reload the flashlight?

Yes. I've added a tutorial message for this in the next update. It's triggered when the flashlight is almost out of power and you have some batteries in your inventory.

> Okay, so I got to the boat part again. As the boat starts moving the minute you step on it, they are standing right on the edge of the thing and it looks like they are standing on air. This time I figured you can actually direct the boat. Not too thrilled about the controls. At least since you are right at the back of the boat you can see the edges of the boat without having to look down, but still it's very hard to see where you are in relation to a wall a lot of time. Sometimes in addition to having no depth reception it's way too dark to see even with the flashlight full of new batteries.

> The boat ride doesn't seem too exciting right now either. There are no any kind of water effects. The boat doesn't make any kind of sounds or splashing effects. If I couldn't constantly see the boat under me it would feel like I was just a ghost floating above water. The player character seems to be standing a bit unnaturally on the boat any way, considering he's trying to control it. Having no sense of your body being affect by the boat rocking would feel more natural if I wasn't required to control the boat.

> None of the physics objects that were falling during the ride made any splashes on the water either. They seemed to go through the water like they were ghost objects. The barrel also didn't seem to float (though I guess it was metal). It created a feeling that I was in no danger at all from the falling debris.

> I do appreciate that the game instantly loads after you die, but it's still kind of frustrating and at the same time very uneventful (there is no sound, no animation, the boat just breaks into pieces in one go). A lot of times I thought I was supposed to land on one of the beaches, but instead I just broke my boat few inches from the land and died. Bunch of times I was going the wrong way either because it was too dark or because it looked like the right direction first, by the time I realized my mistake it was too late and I crashed my boat again. I am not sure if the player should be able to die.

> Right now it feels like an oddity and a huge change from what the game has been so far. It's like a slowed down bullet hell game where your hitbox is not one pixel size like usual.

The raft part definitely needs some iteration. We've gotten a lot of feedback about it. I hope that we can make it work, because I really like how different it is compared to the rest of the game.

**Comment by Au-heppa** [comment]
I keep dying over and over again.

I also just now realized I could move forward in the raft. When I do that I am no longer floating in air but not then I cant see the bounding box anymore. I have no idea where to go right now. 

There was this another beach and I thought maybe I was supposed to disembark there but I kept dying, falling into the water. After I learned that you could move in the raft I tried move to the front part, would fall into the dry land but still die.

I really have no idea what I am supposed to. I am in an intersection and I have tried both directions and the beach. With both directions I can't strafe fast enough and I die before I can dodge the cave wall.

**Comment by Au-heppa** [comment]
Okay, by some miracle I made it through. Not sure what I did different.

But then:

![2015-11-07_00001](https://cloud.githubusercontent.com/assets/5713546/11015829/3fc6135e-8576-11e5-8981-26bf92298c3f.jpg)

Aaaaaaaaaaaaaaaaaaaaaaaaagh!


(Sorry, I don't usually care about trivial graphics problems, but this sort of caught my eye.)

**Comment by Au-heppa** [comment]
I am kind of stuck in the room with the four generators and the hydro decal. There's the G1, G2, G3, and G4 buttons, but they don't do anything. The level also doesn't seem to do anything. There was the password panel with the four buttons marked, but I couldn't find the order, yet. The buttons in the generators also don't seem to be doing anything.

**Comment by aleksijuvani** [comment]
> If the number of photos you have taken affects the game then not being able to save is a problem since it wouldn't be loaded when you continue from another map using the new game menu rather than quick load.

> I would suggest re-enabling the load and save menus and only pushing updates on Sunday evenings or something. Let everyone play tester know that this is the case and that if they continue playing a save file from the previous week they risk having bugs and problems that you have already sorted out. Make it really clear that this is the case. It might feel like it's slowing the progress a little bit, but it's better for the play testers.

I've addressed this in the next update (which we still haven't gotten around to release!) so that all global state is saved when reaching a new map, and when you start a new game in that map, the most recent state is restored. This avoids the usual problems with saved games, but still preserves the player's progress.

> I am kind of stuck in the room with the four generators and the hydro decal. There's the G1, G2, G3, and G4 buttons, but they don't do anything. The level also doesn't seem to do anything. There was the password panel with the four buttons marked, but I couldn't find the order, yet. The buttons in the generators also don't seem to be doing anything.

The order of buttons for the alarm code is deceptively simple. The panel can only be used if the alarm is on though.

You should be able to push the G1, G2, G3 and G4 buttons to turn on the generators. They cannot be used if the alarm is on.

Oh, and you need to override one of the generators in order to flip the circuit breaker.
